doc/modules/tree.rst

```diff
@@ -135,32 +135,26 @@ iris dataset::
 Then we can use Graphviz's ``dot`` tool to create a PDF file (or any other
 supported file type): ``dot -Tpdf iris.dot -o iris.pdf``.
 
-::
+Alternatively, if we have Python module ``graphviz`` installed, we can generate
```

Comment:
Add a mention that if you use conda, you need to install ``graphviz`` and ``graphviz-python``, and if you're not using conda, you need to use your package manger to install the graphviz C library.